tools/clang/test/HLSLFileCheck/pix/DebugVSParameters.hlsl

```diff
@@ -8,7 +8,6 @@
 // CHECK: %CompareToVertId = icmp eq i32 %VertId, 1
 // CHECK: %CompareToInstanceId = icmp eq i32 %InstanceId, 2
 // CHECK: %CompareBoth = and i1 %CompareToVertId, %CompareToInstanceId
-// CHECK: %OffsetMultiplicand = zext i1 %CompareBoth to i32
```

Comment:
I'm curious why test cases such as this and the deleted files were removed.

Reply:
The cases with "phi" in their names were deleted because the pass itself no longer does anything special with PHIs. (It used to add new blocks and edges for all incoming values, for no really good reason, and the extra complexity of the instrumented shader was inducing yet more run-time overhead in the GPU driver.)

This particular line isn't part of the new scheme anymore, and the point of this test file is to validate the checking of the vertex-selection parameters anyway, not the actual per-block debug instrumentation.